src/store/modules/groupApplications.js

```diff
@@ -0,0 +1,104 @@
+import Vue from 'vue'
+import groupApplications from '@/services/api/groupApplications'
+import router from '@/router'
+import { indexById, withMeta, createMetaModule, metaStatuses } from '@/store/helpers'
+
+function initialState () {
+  return {
+    entries: {},
+  }
+}
+export default {
+  namespaced: true,
+  modules: { meta: createMetaModule() },
+  state: initialState(),
+  getters: {
+    get: (state, getters, rootState, rootGetters) => applicationId => {
+      return getters.enrich(state.entries[applicationId])
+    },
+    enrich: (state, getters, rootState, rootGetters) => application => {
+      return application && {
+        ...application,
+        isPending: application.status === 'pending',
+        isDeclined: application.status === 'declined',
+        isAccepted: application.status === 'accepted',
+        isWithdrawn: application.status === 'withdrawn',
+      }
+    },
+    getByGroupId: state => groupId => {
+      return Object.values(state.entries).find(a => a.group === groupId)
+    },
+    allPending: (state, getters) => Object.keys(state.entries).map(getters.get).filter(a => a.isPending).sort(),
```

Comment:
I don't know what it means to just call `sort()` without arguments. I guess you need something like this:  https://github.com//yunity/karrot-frontend/commit/59bb2becb9fe0421438164bfe4f1fff2be7a96fa#diff-196ef378bb7849ac2458a46cc0261277R28